docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -595,6 +595,48 @@ has no DHCP server at all. Therefore, the VM won't have the search domains infor
 reaching a destination using its FQDN is not possible.
 Tracking issue - https://github.com/kubevirt/kubevirt/issues/7184
 
+### passt
+
+`passt` is a new approach for user-mode network that can be used as a simple replacement for Slirp.
+
+```yaml
+kind: VM
+spec:
+  domain:
+    devices:
+      interfaces:
+        - name: red
+          passt: {} # connect using passt mode
+  networks:
+  - name: red
+    pod: {}
+```
+
+#### Requirements:
+1. To get better performance the node should be configured with:
+```
+sysctl -w net.core.rmem_max = 33554432
+sysctl -w net.core.wmem_max = 33554432
+```
+2. To run multiple passt VMs with no explicit ports, the node's `fs.file_max` should be increased
+   (at least ~2^18 per passt with no explicit ports VM):
+```
+sysctl -w fs.file_max = 9223372036854775807
+```
+3. The `vmi.Spec.Domain.Resources.Requests.Memory` of a passt VM with no explicit ports should be at least 2048M.
+
+> **Temporary restrictions:** 
+> 1. `passt` currently only supported as primary network
+> and doesn't allow extra multus networks to be configured on the VM.
```

Comment:
Could you elaborate on the reasoning behind this ? 

Not sure that makes sense on the document: I'm just curious - is it because passt is **very** hungry for memory ?

Reply:
but would that require more memory for each of the pods or not ?... the question may actually be: would we run 1 passt process per iface (as we kind of do for the dhcp server - 1 thread ...) or, would 1 process be able to service all of them ?

This question is purely for my information: the doc is good as is.